### DISCUSSION TITLE
Installer link broken (ARM)

### DISCUSSION THREAD

**Discussion by Chaosghoul** [valinet/ExplorerPatcher#3540] [General]
Hey Folks,
seems like the download link for the ARM version is broken? it's supposed to be this one, but I'm only getting a 404 :( https://github.com/valinet/ExplorerPatcher/releases/latest/download/ep_setup_arm64.exe

Any ideas?

**Comment by Amrsatrio** [comment]
Check the releases section manually for now. Once I tried to make 66.4 a release which should fix that link but MS flagged the x64 version for it so I pullied it back.

**Reply by Chaosghoul** [reply]
Any chance you could post that link in here? I was searching earlier and was too stupid to find it, not a github user

**Reply by pyrates999** [reply]
check here for it: https://github.com/valinet/ExplorerPatcher/releases

There's releases and pre-releases there.

**Reply by Chaosghoul** [reply]
Ah, I had to open up the "Asset" section...
Thanks for helping!

**Comment by Amrsatrio** [comment]
By the way, if you don't mind me asking, which build of ARM64 Windows are you trying to run EP on?

**Reply by Chaosghoul** [reply]
I am on 24H2.
It didn't work right out of the gate, the moment I changed anything with EP, I couldn't open the start menu anymore (even if I just changed the task bar). There was an update waiting in Windows Updater though (some cumulative 24H2 patch of some sort), after which I reinstalled EP and now it is working <3
Edit: OS build 26100.1301 is what I'm currently on
